gossip/src/cluster_info.rs

```diff
@@ -2659,10 +2683,35 @@ impl ClusterInfo {
         Ok(())
     }
 
-    pub fn listen(
+    pub(crate) fn consume(
```

Comment:
nit: functions in the code base that start threads generally start with `start_*_thread`, how about `start_socket_consume_thread()`

Reply:
done